docs/static/persistent-queues.asciidoc

```diff
@@ -33,7 +33,9 @@ The data persisted to disk is not replicated.
 [[configuring-persistent-queues]]
 ==== Configuring persistent queues
 
-To configure persistent queues, specify options in the Logstash <<logstash-settings-file,settings file>>:
+To configure persistent queues, specify options in the Logstash <<logstash-settings-file,settings file>>.
+The settings will be applied to every pipelines, capacity and sizing settings values are to be intended per pipeline.
+If you want to define them for a specific pipeline, use <<multiple-pipelines, `pipelines.yml`>>.
```

Comment:
```suggestion
To configure persistent queues, specify options in the Logstash <<logstash-settings-file,settings file>>.
Settings are applied to every pipeline.

When you set values for capacity and sizing settings, remember that the value you set is applied _per pipeline_ rather than a total to be shared among all pipelines. 

TIP: If you want to define values for a specific pipeline, use <<multiple-pipelines,`pipelines.yml`>>.
```

Reply:
I like the wording, @lucabelluccini  WDYT? Is better, can we commit it and move forward with all the backports with the @karenzone help?

Reply:
The rewording made by @karenzone is even more explicit. I like it.